### ISSUE TITLE
NYC numbers

### ISSUE THREAD

**Issue by anddon333** [covidatlas/coronadatascraper#399]
the total for New York City isn't matching the sum of the 5 counties for 3/25. is this because the city/county sources are different?

**Comment by lazd** [comment]
Yup

![image](https://user-images.githubusercontent.com/201344/77695512-1f8ffa00-6f69-11ea-888e-21468d019627.png)

Left: https://coronavirus.health.ny.gov/county-county-breakdown-positive-cases

Right: https://www1.nyc.gov/assets/doh/downloads/pdf/imm/covid-19-daily-data-summary.pdf

Work for this was done by @aed3 in #281.

@aed3, should we discard `New York City` in favor of a sum of the boroughs from the PDF?

**Comment by aed3** [comment]
I could. Not sure if I should though. The weird thing is the numbers from ny.gov site were updated more recently but have a smaller total, so I don't know which on is more accurate.

**Comment by lazd** [comment]
Yeah @aed3 I don't know which number to take either... There are probably some methodology differences...

**Comment by jzohrab** [comment]
Transferred to https://github.com/covidatlas/li/issues/389

**Comment by jzohrab** [comment]
Transferred to https://github.com/covidatlas/li/issues/390

**Comment by jzohrab** [comment]
Transferred to https://github.com/covidatlas/li/issues/391

**Comment by jzohrab** [comment]
Transferred to https://github.com/covidatlas/li/issues/392

**Comment by jzohrab** [comment]
Transferred to https://github.com/covidatlas/li/issues/393

**Comment by jzohrab** [comment]
Transferred to https://github.com/covidatlas/li/issues/394

**Comment by jzohrab** [comment]
Transferred to https://github.com/covidatlas/li/issues/395

**Comment by jzohrab** [comment]
Transferred to https://github.com/covidatlas/li/issues/396

**Comment by jzohrab** [comment]
Transferred to https://github.com/covidatlas/li/issues/397
